lib/core/src/lib/card-view/components/card-view-keyvaluepairsitem/card-view-keyvaluepairsitem.component.html

```diff
@@ -23,50 +20,44 @@
-            <div class="adf-card-view__key-value-pairs__col">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}</div>
-            <div class="adf-card-view__key-value-pairs__col">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.VALUE' | translate }}</div>
+            <div class="adf-property-col-key">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}</div>
+            <div class="adf-property-col-value">{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.VALUE' | translate }}</div>
         </div>
 
-        <div class="adf-card-view__key-value-pairs__row"
-             *ngFor="let item of values; let i = index">
-            <div class="adf-card-view__key-value-pairs__col">
-                <mat-form-field>
-                    <input matInput
-                           placeholder="{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}"
-                           (blur)="onBlur(item.value)"
-                           [attr.data-automation-id]="'card-'+ property.key +'-name-input-' + i"
-                           [(ngModel)]="values[i].name">
-                </mat-form-field>
-            </div>
-            <div class="adf-card-view__key-value-pairs__col">
-                <mat-form-field>
-                    <input matInput
-                           placeholder="{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.VALUE' | translate }}"
-                           (blur)="onBlur(item.value)"
-                           [attr.data-automation-id]="'card-'+ property.key +'-value-input-' + i"
-                           [(ngModel)]="values[i].value">
-                    <button matSuffix
-                            mat-icon-button
-                            (click)="remove(i)"
-                            class="adf-card-view__key-value-pairs__remove-btn">
-                        <mat-icon>close</mat-icon>
-                    </button>
-                </mat-form-field>
-            </div>
+        <div class="adf-card-view__key-value-pairs__row" *ngFor="let item of values; let i = index">
+            <input
+                matInput
+                class="adf-property-col-key adf-property-value-input"
+                placeholder="{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.NAME' | translate }}"
+                (blur)="onBlur(item.value)"
+                [attr.data-automation-id]="'card-'+ property.key +'-name-input-' + i"
+                [(ngModel)]="values[i].name">
+            
+            <input
+                matInput
+                class="adf-property-col-value adf-property-value-input"
+                placeholder="{{ 'CORE.CARDVIEW.KEYVALUEPAIRS.VALUE' | translate }}"
+                (blur)="onBlur(item.value)"
+                [attr.data-automation-id]="'card-'+ property.key +'-value-input-' + i"
+                [(ngModel)]="values[i].value">
+
+            <button
+                mat-icon-button
+                (click)="remove(i)"
+                class="adf-property-col-delete adf-card-view__key-value-pairs__remove-btn">
```

Comment:
I think you can remove this class adf-card-view__key-value-pairs__remove-btn. It is used only by tests here: 
<img width="1222" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/2ccaaecd-f875-4e79-89de-dbba92960aca">

But instead of this class you can use adf-property-col-delete class in tests.

Reply:
Done